### PR TITLE
TRUNK-5875:Unit test files left in place

### PR DESCRIPTION
**Pull request by jwnasambu**
I worked on the  deleteOnExit() in the BaseContextSensitiveTest to ensure it properly clean-up a directory after running the unit tests.

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5875

### CODE REVIEW

File: api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java [hunk 1]
```diff
@@ -98,7 +98,7 @@
 import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
 import org.springframework.transaction.annotation.Transactional;
 import org.xml.sax.InputSource;
-
+import org.apache.commons.io.FileUtils;
```
Comment (ibacher) [inline api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java:101]: This should go with the imports above, ideally just before `import org.apache.commons.io.IOUtils;`

**Comment by tendomart** [comment]
@ibacher do you think if she applied a Eclipse Code Formatter  , it would be of help ?

**Comment by HerbertYiga** [comment]
@tendomart   mvn clean package also helps on the unwanted code formats

File: api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java (hunk 1 above)
Comment (jwnasambu) [reply api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java:101]: @ibacher Thanks let me fix it.

**Comment by jwnasambu** [comment]
@tendomart  and @HerbertYiga I really appreciate you comments

**Pull request closed without merging**